src/work-allocation/containers/task-action/task-action-container.component.spec.ts

```diff
@@ -3,50 +3,56 @@ import { HttpClientTestingModule } from '@angular/common/http/testing';
 import { Component, Input, ViewChild } from '@angular/core';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
 import { FormsModule } from '@angular/forms';
-import { ActivatedRoute } from '@angular/router';
+import { ActivatedRoute, Router } from '@angular/router';
 import { RouterTestingModule } from '@angular/router/testing';
-import { LoadingService, PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
+import { PaginationModule } from '@hmcts/ccd-case-ui-toolkit';
 import { SessionStorageService } from '@hmcts/ccd-case-ui-toolkit/dist/shared/services';
-import { Observable, of } from 'rxjs';
-
+import { Observable, throwError } from 'rxjs';
 import { TaskListComponent } from '..';
```

Comment:
Minor comment - incorrect import ordering

Reply:
Scope of "Removing MVP" is not finetuning previously developed code , explained in detail on another comment.